osquery/tables/applications/posix/carbon_black.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <osquery/tables.h>
+#include <osquery/core.h>
+#include <osquery/filesystem.h>
+
+#include <boost/filesystem.hpp>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/ini_parser.hpp>
+#include <boost/algorithm/string/replace.hpp>
+#include <boost/algorithm/string/predicate.hpp>
+
+namespace fs = boost::filesystem;
+
+namespace osquery {
+namespace tables {
+
+// Path to the Carbon Black sensor id file
+#define kCbSensorIdFile "/var/lib/cb/sensor.id"
+// Path to the Carbon Black sensor settings file
+#define kCbSensorSettingsFile "/var/lib/cb/sensorsettings.ini"
+// Path to Carbon Black direcotry
+#define kCbDir "/var/lib/cb/"
+
+// Get the Carbon Black sensor ID
+void getSensorId(Row& r) {
+    std::string file_contents;
```

Comment:
Failing the code audit because this file wasn't picked up by `make format_master`.